### PR TITLE
Issue/headers fix

### PR DESCRIPTION
**Pull request by zakagan**

### CODE REVIEW

File: dnaStreaming/config.py [hunk 1]
```diff
@@ -5,6 +5,8 @@
 import os
 
 # Python3 has FileNotFoundError defined. Python2 does not.
+from dnaStreaming.services.credentials_service import get_authentication_headers
```
Comment (rapkyt) [inline dnaStreaming/config.py]: I'd swap line 7 and 8, except for that, Approved!

File: dnaStreaming/config.py [hunk 2]
```diff
@@ -48,6 +52,25 @@ def _validate(self):
         if not os.access(self.customer_config_path, os.R_OK):
             raise Exception('Encountered permission problem reading file from path \'{}\'.'.format(self.customer_config_path))
 
+    def get_headers(self):
+        if self.headers:
+            return self.headers
+        else:
+            self.headers = get_authentication_headers(self)
+            return self.headers
```
Comment (cristianmoradj) [inline dnaStreaming/config.py:59]: What about changing this to:
```
if self.headers is None:
    self.headers = get_authentication_headers(self)
return self.headers
```

**Comment by cristianmoradj** [comment]
Appoved!!

File: dnaStreaming/test/test_run_docker.sh [hunk 3]
```diff
@@ -26,7 +26,7 @@ run_docker() {
 
   docker run \
   --name=$NAME \
-  -e SERVICE_ACCOUNT_ID=$SERVICE_ACCOUNT_ID \
+  -e USER_KEY=$USER_KEY \
   -e SUBSCRIPTION_ID=$SUBSCRIPTION_ID \
   -e CREDENTIALS_URI="https://extraction-api-dot-djsyndicationhub-$ENV.appspot.com/alpha/accounts/streaming-credentials" \
```
Comment (dj-dna-andersend) [inline dnaStreaming/test/test_run_docker.sh]: update this to EXTRACTION_API_HOST

Comment (dj-dna-andersend) [reply dnaStreaming/test/test_run_docker.sh]: and remove everything after the host in the URI itself

**Review by dj-dna-andersend — APPROVED** [review]

**Pull request closed without merging**